CHANGELOG.md

```diff
@@ -6,7 +6,6 @@ FEATURES
 
 IMPROVEMENTS
 * export command now writes current validator set for Tendermint
-* [tests] Application module tests now use a mock application
```

Comment:
I don't think you meant to delete this here.

Reply:
Thanks yeah I don't know how this happened